ironmon_tracker/screens/TrackerScreen.lua

```diff
@@ -856,7 +866,8 @@ function TrackerScreen.drawMovesArea(pokemon, opposingPokemon)
 
 		-- DRAW MOVE EFFECTIVENESS
 		if Options["Show move effectiveness"] and Battle.inBattle and showEffectiveness then
-			local enemyTypes = Program.getPokemonTypes(not Tracker.Data.isViewingOwn)
+			local enemyTypes = opposingPokemon.types
+			--Program.getPokemonTypes(not Tracker.Data.isViewingOwn)
```

Comment:
Not sure what's going on here. Originally the `Program.getPokemonTypes()` was necessary to get dynamic types of the "other" Pokemon. This is important in the case of Color Change or Transform.

Is this dynamic check made elsewhere? If so, I can't quite tell where